src/System.Management.Automation/engine/LanguagePrimitives.cs

```diff
@@ -14,6 +14,7 @@
 using System.Management.Automation.Internal;
```

Comment:
@vexx32, your last commit had 2 failures in `PowerShell-CI-windows`
Validate 'Save-Help -DestinationPath for all PowerShell modules..Validate Save-Help for the 'Microsoft.PowerShell.Diagnostics' module
```powershell
Expected 'Microsoft.PowerShell.Diagnostics_ca046f10-ca64-4740-8ff9-2565dba61a4f_en-US_helpcontent.cab', because Save-Help for  should download 'Microsoft.PowerShell.Diagnostics_ca046f10-ca64-4740-8ff9-2565dba61a4f_en-US_helpcontent.cab', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\Help\UpdatableHelpSystem.Tests.ps1: line 333
333:     $compressedFile | Should -Be $expectedCompressedFile -Because "Save-Help for $module should download '$expectedCompressedFile'"
```
method conversion.Correctly casts 100n to value 100 as type [int]
```powershell
Expected '100', but got $null.
at <ScriptBlock>, D:\a\1\s\test\powershell\Language\Parser\Conversions.Tests.ps1: line 510
510:         $Result | Should -Be $Value
```